dart/lib/src/sentry_options.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:http/http.dart';
+import 'package:meta/meta.dart';
+
+import 'protocol.dart';
+import 'utils.dart';
+
+/// Sentry SDK options
+class SentryOptions {
+  /// Default Log level if not specified Default is DEBUG
+  static final SeverityLevel defaultDiagnosticLevel = SeverityLevel.debug;
+
+  /// The DSN tells the SDK where to send the events to. If this value is not provided, the SDK will
+  ///  just not send any events.
+  final String dsn;
+
+  /// Contains [Event] attributes that are automatically mixed into all events
+  /// captured through this client.
+  ///
+  /// This event is designed to contain static values that do not change from
+  /// event to event, such as local operating system version, the version of
+  /// Dart/Flutter SDK, etc. These attributes have lower precedence than those
+  /// supplied in the even passed to [capture].
+  final Event environmentAttributes;
```

Comment:
ideally we'd follow already defined names https://github.com/getsentry/sentry-java/blob/main/sentry/src/main/java/io/sentry/SentryOptions.java#L117-L122
if they are about the same attribute

Reply:
renamed to `environment`

Reply:
reverted to `environmentAttributes`